### PR TITLE
Sf2 cov

### PR DESCRIPTION
**Pull request by wmFrank**
add a config file to define the code coverage range

### CODE REVIEW

File: .core-coveragerc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[run]
+omit =
+    setup.py
+    tests/*
```
Comment (alex-petrenko) [inline .core-coveragerc:4]: Aren't tests covered by tests themselves? What's the purpose of removing tests?

File: .core-coveragerc [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    sample_factory/runner/runs/*
+    sample_factory/huggingface/*
```
Comment (alex-petrenko) [inline .core-coveragerc:6]: I think we can keep huggingface and PBT. We should only remove folders that we can't test in CI such as doom or isaacgym. But PbT and huggingface we can write tests for. So coverage below 100% should reflect that! :)

File: Makefile [hunk 3]
```diff
@@ -56,3 +56,10 @@ test:
 test-cov:
 	pytest --cov=./ -v
 # ; echo "Tests finished. You might need to type 'reset' and press Enter to fix the terminal window"
+
+# Run code coverage test
```
Comment (alex-petrenko) [inline Makefile:60]: :+1:

**Review by alex-petrenko — CHANGES_REQUESTED** [review]
Please see my minor comments!

File: .core-coveragerc (hunk 1 above)
Comment (wmFrank) [reply .core-coveragerc:4]: We write tests for sample-factory code, but we do not write tests for our tests.

Comment (wmFrank) [reply .core-coveragerc:4]: Correct me if I am wrong.

File: .core-coveragerc (hunk 2 above)
Comment (wmFrank) [reply .core-coveragerc:6]: OK, got it.

**Review by alex-petrenko — APPROVED** [review]

**Pull request merged**